src/utils/wallet.js

```diff
@@ -156,6 +157,14 @@ export const groomingWalletData = async (user, globalProps) => {
 
   walletData.estimatedValue = totalSteem * pricePerSteem + totalSbd;
 
+  const ppSbd = await getCurrencyTokenRate('usd', 'sbd');
```

Comment:
@mistikk Can you check this and possibly fix? I added currency rate for STEEM and SBD to get correct estimated value, but it is only getting `usd` rate and when user changes currency to EUR, it shows completely wrong estimated values.